lms/djangoapps/courseware/tests/test_submitting_problems.py

```diff
@@ -268,55 +268,33 @@ def add_grading_policy(self, grading_policy):
         self.update_course(self.course, self.student_user.id)
         self.refresh_course()
 
-    def get_grade_summary(self):
+    def get_course_grade(self):
```

Comment:
👍 Really happy to see these multiple methods collapsed into a single entry point